src/lib/modules/webserver/server.js

```diff
@@ -71,36 +67,11 @@ class Server {
       next();
     });
 
-    this.app.use(helmet.noCache());
-    this.app.use(cors());
```

Comment:
I believe those 2 are not needed anymore as there is not API behind.
Only the dapp stays.